opentelemetry-api/src/opentelemetry/trace/__init__.py

```diff
@@ -99,6 +100,46 @@ def get_context(self) -> 'SpanContext':
         """
 
 
+class TraceOptions(int):
+    """A bitmask that represents options specific to the trace.
+
+    The only supported option is the "recorded" flag (``0x01``). If set, this
+    flag indicates that the trace may have been recorded upstream.
+
+    See the `W3C Trace Context`_ spec for details.
+
+    .. _W3C Trace Context: https://www.w3.org/TR/trace-context/#trace-flags
+    """
+    DEFAULT = 0x00
+    RECORDED = 0x01
+
+    @classmethod
+    def get_default(cls) -> 'TraceOptions':
+        return cls(cls.DEFAULT)
+
+
+DEFAULT_TRACEOPTIONS = TraceOptions.get_default()
```

Comment:
One thing we need to decide is whether we want these as module or class level members.

Reply:
I don't have a strong preference, but we'd have to do the assignment (e.g. `TraceOptions.DEFAULT = TraceOptions.get_default`) outside the class definition anyway.